core/iwasm/include/wasm_c_api.h

```diff
@@ -181,24 +181,28 @@ typedef union MemAllocOption {
 struct wasm_config_t {
     mem_alloc_type_t mem_alloc_type;
     MemAllocOption mem_alloc_option;
-    bool linux_perf_support;
+#if WASM_ENABLE_LINUX_PERF != 0
```

Comment:
Don't add macro here, like RuntimeInitArgs in wasm_export.h, the embedder may use the header file and libiwasm.so/libvmlib.a in his project, and doesn't define the macro to compile WAMR again.